src/main/java/net/mcreator/ui/dialogs/wysiwyg/ButtonDialog.java

```diff
@@ -37,22 +38,22 @@ public ButtonDialog(WYSIWYGEditor editor, @Nullable Button button) {
 		setModal(true);
 		setSize(480, 200);
 		setLocationRelativeTo(editor.mcreator);
-		setTitle("Add button");
+		setTitle(L10N.t("dialog.gui.button_add_title"));
 		JTextField nameField = new JTextField(20);
 		JPanel options = new JPanel();
 		options.setLayout(new BoxLayout(options, BoxLayout.PAGE_AXIS));
 
 		if (button == null)
 			add("North", PanelUtils.centerInPanel(
-					new JLabel("After you click OK, right-click with the mouse to change button width")));
+					L10N.label("dialog.gui.button_change_width")));
 		else
 			add("North", PanelUtils.centerInPanel(
-					new JLabel("To resize the button, use move tool and right-click with the mouse when moving")));
+					L10N.label("dialog.gui.button_resize")));

```

Comment:
This key seems to be missing in the translations file.

Reply:
Please verify for presence of all other keys too @Max094Reikeb

Reply:
Sure! I'll do it ASAP

Reply:
Problem fixed!